modules/Core/includes/endpoints/SetGroupFromDiscordIdEndpoint.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * @param int $discord_user_id The Discord ID of the NamelessMC user to edit
+ * @param int $discord_id The Discord ID fo the NamelessMC group to apply as their primary group
+ * 
+ * @return string JSON Array
+ */
+class SetGroupFromDiscordIdEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'setGroupFromDiscordId';
+        $this->_module = 'Core';
+        $this->_description = 'Set a NamelessMC user\'s primary group from a provided Discord role ID';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.